public/index.html

```diff
@@ -2,16 +2,30 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>CavaTable</title>
+  <meta charset="utf-8" />
+  <title>CavaTable - Photos</title>
+  <meta name="description" content="description">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro">
+  <style>
+    .photo-gallery-header {
```

Comment:
I'm not sure if you should style stuff in html inline. Maybe just use the css file.